modules/Microsoft.Storage/static-website/1.0/scripts/enable-storage-static-website.ps1

```diff
@@ -0,0 +1,16 @@
+$ErrorActionPreference = 'Stop'

```

Comment:
thinking more about this as a module, this isn't terribly useful in that I just end up with a hello world website...  though on the flip side I can't think of a good way to populate it through the module...  Any thoughts on how we could use this and have the user provide "real" files for the site via a module?  via loadTextContent() maybe?  or zip as base64 content?